RoadMap_SwiftUI/RoadMap_SwiftUI/Helper/CustomToggleStyle.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  CustomToggleStyle.swift
+//  RoadMap_SwiftUI
+//
+//  Created by Артур Щукин on 27.01.2023.
+//
```

Comment:
```suggestion
//
//  CustomToggleStyle.swift
//  RoadMap_SwiftUI
//
//  Created by Артур Щукин on 27.01.2023.
//
```
не отработал форматер